pkg/util/util_test.go

```diff
@@ -33,7 +33,7 @@ func TestFlatten(t *testing.T) {
 	}
 
 	for index, testData := range testDataSlice {
-		t.Run(string(index), func(t *testing.T) {
+		t.Run(fmt.Sprint(index), func(t *testing.T) {
```

Comment:
heading says version upgrade, how are these changes related?

Reply:
I'll create another PR to address this. Revert it.

Reply:
thanks